client/src/components/SearchBar.js

```diff
@@ -0,0 +1,193 @@
+import { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import axios from 'axios'
+
+
+export default function SearchBar({ placeholderText, onCreateIngredient }) {
+
+    const [searchQuery, setSearchQuery] = useState('')
+    const [fetchedIngredients, setFetchedIngredients] = useState([])
+    const [ingredient, setIngredient] = useState({})
+
+    const [isError, setIsError] = useState(false)
+
+    const getQueryValue = (event) => {
+        const value = event.target.value
+        const query = value.toLowerCase()
+        setSearchQuery(query)
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if (searchQuery.length >= 3 && ingredient.length !== 0 && isError !== true) {
```

Comment:
I think you can shorten the last query to `!isError`as `isErrror`is a boolean
```suggestion
        if (searchQuery.length >= 3 && ingredient.length !== 0 && !isError) {
```

Reply:
Thank you for the great tipp!